cmd/main.go

```diff
@@ -31,8 +31,5 @@ func main() {
 	}
 
```

Comment:
you should handle sys calls somewhere. you should have a health check server as well